massa_proxy/apps/massa_proxy/priv/protos/frontend/google/api/auth.proto

```diff
@@ -0,0 +1,228 @@
+// Copyright 2020 Google LLC
```

Comment:
In fact, this only exists to avoid unnecessary compilations at run time, so it is included at compile time. Look at here https://github.com/eigr/massa/blob/a03e074cebd819fe2b30d43bb53110945052c421/massa_proxy/apps/massa_proxy/lib/massa_proxy/server.ex#L20